frontend/package-lock.json

```diff
@@ -3721,9 +3721,9 @@
       "integrity": "sha512-/LLMVyas0ljjAtoYiPqYiL8VWXzUUdThrmU5+n20DZv+a+ClRoevUzw5JxU+Ieh5/c87ytoTBV9G1FiKfNJdmg=="
     },
     "node_modules/@next/env": {
-      "version": "14.2.22",
-      "resolved": "https://registry.npmjs.org/@next/env/-/env-14.2.22.tgz",
-      "integrity": "sha512-EQ6y1QeNQglNmNIXvwP/Bb+lf7n9WtgcWvtoFsHquVLCJUuxRs+6SfZ5EK0/EqkkLex4RrDySvKgKNN7PXip7Q=="
+      "version": "14.2.12",
```

Comment:
@kaloster somehow this is not using 14.2.25. Maybe that's why the check is failing 😄 Thank you!